example/scripts/generate_ingest/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@cumulus/example/generate_ingest",
+  "version": "4.0.0",
+  "description": "Script to generate test data for scaled ingest",
```

Comment:
What about a `"private": true,` line?

Reply:
Yes, we probably don't want to lerna publish this.

Reply:
b79a52f99